tests/conftest.py

```diff
@@ -9,9 +9,18 @@
 
 from aiida_icon.calculations import IconCalculation
 
+# pytest configuration
+
 pytest_plugins = ["aiida.tools.pytest_fixtures"]
 
 
+def pytest_configure(config):
+    config.addinivalue_line("markers", "requires_icon: marks test to require icon installation")
+
+
+# Fixtures
+
+
```

Comment:
this should go into the pyproject.toml instead.

```
[tool.pytest.ini_options]
markers = [
  'requires_icon: marks test to require icon installation'
]
```

Reply:
done!